mlflow/store/tracking/abstract_store.py

```diff
@@ -271,3 +271,20 @@ def log_batch(self, run_id, metrics, params, tags):
         :return: None.
         """
         pass
+
+    @abstractmethod
+    def record_logged_model(self, run_id, mlflow_model):
```

Comment:
Can we add the @mlflow.utils.experimental annotation here?

Reply:
yeah, good point.